pkg/plugin/v1/scaffolds/internal/templates/crd/crd.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crd
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+
+	"github.com/kr/text"
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+var _ file.Template = &CRD{}
+
+// CRD scaffolds a manifest for CRD sample.
+type CRD struct {
+	file.TemplateMixin
+	file.ResourceMixin
+
+	CRDVersion string
+}
+
+// SetTemplateDefaults implements input.Template
+func (f *CRD) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = filepath.Join("config", "crd", "bases", fmt.Sprintf("%s_%%[plural].yaml", f.Resource.Domain))
+	}
+	f.Path = f.Resource.Replacer().Replace(f.Path)
+
+	f.IfExistsAction = file.Error
+
+	if f.CRDVersion == "" {
+		f.CRDVersion = "v1"
+	} else if f.CRDVersion != "v1" && f.CRDVersion != "v1beta1" {
+		return errors.New("CRD version must be either 'v1' or 'v1beta1'")
```

Comment:
```suggestion
		return errors.New("the CRD version value must be either 'v1' or 'v1beta1'")
```
otherwise will not pass in the sanity check. 
 

```
Error messages do not begin with lower case:
sdk/internal/plugins/helm/v1/scaffolds/templates/crd/crd.go:48:		return errors.New("CRD version must be either 'v1' or 'v1beta1'")
```